ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,946 @@
+-  `Delete the API key, service account, Kafka topics, Kafka cluster, environment,
+   and the log files`_
+
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+-  Access to a |ccloud| cluster. See `Create a Cluster in Confluent Cloud
+   <https://docs.confluent.io/current/cloud/clusters/create-cluster.html>`__
+
+-  Local `install of Confluent Cloud CLI
+   <https://docs.confluent.io/current/cloud/cli/install.html>`__ (v1.7.0 or later)
+
+-  |ccloud| user credentials saved in ``~/.netrc``. (Use ``ccloud login --save``
+   when logging in to the |ccloud| CLI. The ``--save`` flag will save your login
+   credentials to the ``~/.netrc`` file.)
+
+-  `Docker <https://docs.docker.com/get-docker/>`__ and `Docker Compose
+   <https://docs.docker.com/compose/install/>`__ for the local |kconnect| worker
+
+-  .. include:: ../../ccloud/docs/includes/prereq_timeout.rst
+
+-  `mvn <https://maven.apache.org/install.html>`__ installed on your host
+
+-  `jq <https://github.com/stedolan/jq/wiki/Installation>`__ installed on your host
+
+
+Confluent Cloud Promo Code
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first 20 users to sign up for `Confluent Cloud
+<https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+and use promo code ``C50INTEG`` will receive an additional $50 free usage
+(`details
+<https://www.confluent.io/confluent-cloud-promo-disclaimer/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__).
+
+
+Run the demo
+------------
+
+.. note::
+
+   - This example uses real resources in |ccloud|, and it creates and deletes
+     topics, service accounts, API keys, and ACLs.
+
+   - The values for certain variables, including your API key, |ak|
+     cluster, and environment will be unique and may not match the output
+     exactly.
+
+To run this demo, complete the following steps:
+
+#. Change directory to the ``.sh`` file for |ccloud| CLI.
+
+   .. code-block:: bash
+
+      cd examples/ccloud/beginner-cloud/
+
+#. Run the following command to start the demo:
+
+   .. code-block:: bash
+
+         ./start.sh
+
+The demo will carry out the following workflow.
+
+Create a new Confluent Cloud environment and specify it as the default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following command to create a new |ccloud| environment
+   ``demo-script-env``:
+
+   .. code-block:: bash
+
+      ccloud environment create demo-script-env -o json
+
+#. Verify your output resembles:
+
+   .. code-block:: text
+
+      {
+        "id": "env-5qz2q",
+        "name": "demo-script-env"
+      }
+
+#. Specify ``env-5qz2q`` as the active environment by running the following
+   command:
+
+   .. code-block:: bash
+
+       ccloud environment use env-5qz2q
+
+   You should see output similar to:
+
+   .. code-block:: text
+
+      Now using "env-5qz2q" as the default (active) environment.
```

Comment:
@ybyzek How about we strip the "default" here and just say: **Now using "env-5qz2q" as the active environment.** I put a comment/suggestion in the Friction Log for this one. If this is cool, I'll make the updates anywhere else this may occur in the tutorial.